dascore/proc/align.py

```
@@ -0,0 +1,315 @@
+    max_ends = start + n_samples
+    end = np.where(max_ends > output_size, output_size, max_ends)
+    return (start, end)
+
+
+def _calculate_shift_info(
+    mode,
+    shifts,
+    dim,
+    dim_axis,
+    patch_shape,
+    reverse,
```

Ruff (0.13.3)

Unused function argument: `reverse`

(ARG001)